### ISSUE TITLE
Feature request: "Move all pending items to top" automatically

### ISSUE THREAD

**Issue by Dizzap** [martinstoeckli/SilentNotes#122]
Would it be possible to change this function to happen automatically, when you mark an item as done? Either generally or just in shopping mode.  
When I have a very long shopping list, it's quite hard to find unchecked items, and I either have to constantly switch off shopping mode to sort them (since the menu is disabled), or just delete them right away and not bother with shopping mode.  
It'd basically work the same way as Google Keep does - you mark item as done, and it just disappears to the bottom.

Let me know what you think.
Cheers,
Jakub

**Comment by martinstoeckli** [comment]
Yes this would of course make sense, and it is something that was requested before by other users. Maybe it would be preferable to make it a global option in the settings menu, I suppose it is something a user wants for all or none of the to-do lists. As you said, reacting to the check event would play well with the shopping mode.

**Comment by pkoevesdi** [comment]
I really want to affirm this wish! I'm used to this behaviour from google keep, which I want to switch away from.
